src/config.js

```diff
@@ -39,6 +39,8 @@ module.exports = Object.assign({
   pintrestApiUrl: process.env.PINTREST_API_URL || 'https://api.pinterest.com/v1/',
   pintrestEnabled: process.env.PINTREST_ACCESS_TOKEN !== undefined && pinterestActivated,
   ltiActivated,
+  auth0ClientID: '25sRFeGU4iqARHylaIArs8xzTo1I4jn9',
+  auth0Domain: 'ndla.eu.auth0.com',
```

Comment:
Disse to burde vel være variabler som man får inn, og ikke hardkodet her? Vi trenger ikke publisere dette på github egentlig (selv om en klient-id er med i requesten)